packages/core/src/components/Text/__stories__/Text.stories.module.scss

```diff
@@ -1,31 +1,9 @@
-@mixin spacing {
-  padding: var(--sb-spacing-small);
-}
-
-.primaryBackground {
-  background-color: var(--sb-primary-color);
-}
-
-.invertedBackground {
-  background-color: var(--sb-inverted-color-background);
-}
-
-.smallRectangle {
-  @include spacing;
-  width: 150px;
-}
-
-.mediumRectangle {
-  @include spacing;
-  width: 420px;
-}
-
 .textBoxContainer {
```

Comment:
This is needed for the do's and don't but can be replaced with an image in the future